src/verifiers/documentStatus/didSignedDocumentStatus/didSignedDocumentStatus.test.ts

```diff
@@ -1,33 +1,224 @@
+        Object {
+          "data": Object {
+            "details": Object {
+              "issuance": Array [
+                Object {
+                  "did": "did:ethr:0xE712878f6E8d5d4F9e87E10DA604F9cB564C9a89",
+                  "issued": true,
+                },
+              ],
+            },
+            "issuedOnAll": true,
+            "revokedOnAny": false,
+          },
+          "name": "OpenAttestationDidSignedDocumentStatus",
+          "status": "VALID",
+          "type": "DOCUMENT_STATUS",
+        }
+      `);
+    });
+    it("should pass for documents using `DID-DNS` and is correctly signed", async () => {
+      whenPublicKeyResolvesSuccessfully();
+      const res = await OpenAttestationDidSignedDocumentStatus.verify(documentDnsDidSigned, options);
+      expect(res).toMatchInlineSnapshot(`
+        Object {
+          "data": Object {
+            "details": Object {
+              "issuance": Array [
+                Object {
+                  "did": "did:ethr:0xE712878f6E8d5d4F9e87E10DA604F9cB564C9a89",
+                  "issued": true,
+                },
+              ],
+            },
+            "issuedOnAll": true,
+            "revokedOnAny": false,
+          },
+          "name": "OpenAttestationDidSignedDocumentStatus",
+          "status": "VALID",
+          "type": "DOCUMENT_STATUS",
+        }
+      `);
+    });
+    it("should fail when revocation block is missing", async () => {
+      whenPublicKeyResolvesSuccessfully();
+      const res = await OpenAttestationDidSignedDocumentStatus.verify(documentDidObfuscatedRevocation, options);
+      expect(res).toMatchInlineSnapshot(`
+        Object {
+          "data": [Error: revocation block not found for an issuer],
+          "name": "OpenAttestationDidSignedDocumentStatus",
+          "reason": Object {
+            "code": 1,
+            "codeString": "UNEXPECTED_ERROR",
+            "message": "revocation block not found for an issuer",
+          },
+          "status": "ERROR",
+          "type": "DOCUMENT_STATUS",
+        }
+      `);
+    });
+    it("should fail when revocation is not set to NONE (for now)", async () => {
+      whenPublicKeyResolvesSuccessfully();
+      const res = await OpenAttestationDidSignedDocumentStatus.verify(documentDidCustomRevocation, options);
+      expect(res).toMatchInlineSnapshot(`
+        Object {
+          "data": Object {
+            "details": Object {
+              "issuance": Array [
+                Object {
+                  "did": "did:ethr:0xE712878f6E8d5d4F9e87E10DA604F9cB564C9a89",
+                  "issued": true,
+                },
+              ],
+            },
+            "issuedOnAll": true,
+            "revokedOnAny": true,
+          },
+          "name": "OpenAttestationDidSignedDocumentStatus",
+          "status": "INVALID",
+          "type": "DOCUMENT_STATUS",
+        }
+      `);
+    });
+    it("should fail when proof is missing", async () => {
```

Comment:
actually in this case it should be skipped, because the test functions wont let this one run (there is no proof block at all)

Reply:
the error thrown is the one tested by `should fail when corresponding proof to key is not found in proof`

Reply:
Right, but this is the unit test when the verify is forcibly ran haha